### PR TITLE
Fix CI post-merge PHP 8

### PR DESCRIPTION
**Pull request by vincentchalamon**

### CODE REVIEW

**Comment by Nyholm** [comment]
We can maybe make this simpler. We can try multiple PHP versions on SF5.x only. That will reduce the number of jobs.

**Comment by vincentchalamon** [comment]
I propose to test Symfony 3.4 and 4.4 only on PHP 7.1, and without the lowest composer dependencies, so we keep some tests on those Symfony versions.

Symfony 5.0 will be tested on PHP 7.2, 7.3, 7.4 and 8.0 with latest and lowest dependencies.

WDYT @Nyholm?

**Comment by vincentchalamon** [comment]
OK this matrix is not working. I will only test Symfony 5 on PHP 7.2+

Should I test on Symfony 5.0, 5.1 or 5.2?

**Comment by Nyholm** [comment]
Sound good!

**Comment by vincentchalamon** [comment]
@Nyholm Just need your help to fix the coverage upload, maybe some permissions or anything similar would be missing.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -72,15 +59,15 @@ jobs:
...
       - name: Run tests with coverage
         if: matrix.coverage
         run: vendor/bin/phpunit --coverage-text --coverage-clover=build/coverage.xml
 
       - name: Upload coverage
-        if: matrix.coverage
+        if: matrix.coverage && matrix.dependency == ''
         run: |
           wget https://scrutinizer-ci.com/ocular.phar
           php ocular.phar code-coverage:upload --format=php-clover build/coverage.xml
```
Comment (Nyholm) [inline .github/workflows/ci.yml]:
```suggestion
          php ocular.phar code-coverage:upload --repository Nyholm/symfony-bundle-test --format=php-clover build/coverage.xml
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -72,15 +59,15 @@ jobs:
...
         if: matrix.coverage
         run: vendor/bin/phpunit --coverage-text --coverage-clover=build/coverage.xml
 
       - name: Upload coverage
-        if: matrix.coverage
+        if: matrix.coverage && matrix.dependency == ''
         run: |
           wget https://scrutinizer-ci.com/ocular.phar
-          php ocular.phar code-coverage:upload --format=php-clover build/coverage.xml
+          php ocular.phar code-coverage:upload --repository Nyholm/symfony-bundle-test --format=php-clover build/coverage.xml
```
Comment (Nyholm) [inline .github/workflows/ci.yml]:
```suggestion
          php ocular.phar code-coverage:upload --repository g/Nyholm/symfony-bundle-test --format=php-clover build/coverage.xml
```

Comment (Nyholm) [reply .github/workflows/ci.yml]: Typo..

**Review by Nyholm — COMMENTED** [review]
Let's remove the code coverage and the bandges of quality from scrutinizer. It has not been used since the repository was under my username. 

Also make sure that we test on SF 3.4, SF 4.4 and SF5.x at least once.

**Comment by vincentchalamon** [comment]
@Nyholm Symfony 3.4 and 4.4 are tested on PHP 7.2 with latest dependencies. Symfony 5.0 is tested on PHP 7.2, 7.3, 7.4 and 8.0 with lowest and latest dependencies.

I've removed the coverage feature from the CI, and the related badges.

You should check to disable travis from CI.

**Comment by Nyholm** [comment]
Excellent. I think all looks good. 

Thank you for your work!

Let me know if you need anything else for hacktoberfest

**Pull request merged**